resources/guides/ta_guide.md

```diff
@@ -0,0 +1,99 @@
+---
+id: teaching assistant
+title: How To Become a Teaching Assistant
+sidebar_label: How To Become a Teaching Assistant
+slug: /guides/ta_guide
+---
+
+_Last updated on 2023/09/07_
+
+## What is a TA?
+
+A TA is a unionized student position where students can work with instructors to run University of Windsor courses. Instructors could ask for a TA's help in a variety of ways:
+
+- Holding Office Hours 
+- Grade student assessments
+- Proctor Exams
+- Taking notes or answering questions during classes
+    
+A TA refers to an undergraduate student while GA (Graduate Assistant) refers to Graduate students
+
+## Why should you become a TA?
+
+Being a TA provides many benefits:
+
+- Work experience with a subject of interest
+- Create connections with Professors and Instructors
+- Paid Position
```

Comment:
Should we include an approximate hourly rate? I recall $20CAD/hr, max 100 hours but I don't know how often that changes, if at all.

Reply:
I can include the pay scale for TA I and TA II as of the current agreement, I will also add a link to find the latest collective agreement so they can see any changes made in the future.